packages/tailwindcss/src/test-utils/run.ts

```diff
@@ -6,9 +6,10 @@ export async function compileCss(css: string, candidates: string[] = [], options
   return optimizeCss(build(candidates)).trim()
 }
 
-export async function run(candidates: string[]) {
+export async function run(candidates: string[], { optimize = true } = {}) {
```

Comment:
Not a big fan of having to add an object with `optimize` to every call site. How do you feel about this?

I’m wondering if we:
- Should make this the default (or is the number of `run()` calls that do need `optimize: true` even larger than this subset?
- Should instead add a second `runOptimized` export instead of adding a boolean variable?

Reply:
Also not a fan but not enough to really care. I think unoptimized is *not* the default. Pretty sure more tests require an optimized CSS result than not but I don't have a good idea of what the function should be called.

Reply:
Funny thing is, I did use a new method initially (https://github.com/tailwindlabs/tailwindcss/pull/14704/commits/14817cba32d36097fff0bc886302a67e69d21624) but `run` vs `runOptimized` or vs `build` all felt wrong, so did a fallback here because typically you run it without.

You could argue that we just don't run the tests through Lightning CSS (which will make the tests bigger but faster).